guide/14-deep-learning/add_custom_nlp_models_using_model_extension.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p><code>each parameter that your</code> should be replaced with <code>customizable parameter that user can alter</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2000/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>